framework/decode/custom_vulkan_struct_to_json.cpp

```diff
@@ -418,5 +418,48 @@ void FieldToJson(nlohmann::ordered_json&                                     jda
     FieldToJson(jdata["pData"], &pData->pData, options);
 }
 
+void FieldToJson(nlohmann::ordered_json&                                  jdata,
+                 const Decoded_VkIndirectExecutionSetCreateInfoEXT* const pData,
+                 const util::JsonOptions&                                 options)
+{
+    FieldToJson(jdata["type"], pData->decoded_type, options);
+    switch (pData->decoded_type)
+    {
+        case VK_INDIRECT_EXECUTION_SET_INFO_TYPE_PIPELINES_EXT:
+            FieldToJson(jdata["info"], pData->info->pPipelineInfo, options);
+            break;
+        case VK_INDIRECT_EXECUTION_SET_INFO_TYPE_SHADER_OBJECTS_EXT:
+            FieldToJson(jdata["info"], pData->info->pShaderInfo, options);
+            break;
+        default:
+            break;
+    }
+}
+
+void FieldToJson(nlohmann::ordered_json&                               jdata,
+                 const Decoded_VkIndirectCommandsLayoutTokenEXT* const pData,
+                 const util::JsonOptions&                              options)
+{
+    FieldToJson(jdata["type"], pData->decoded_type, options);
+    switch (pData->decoded_type)
+    {
+        case VK_INDIRECT_COMMANDS_TOKEN_TYPE_PUSH_CONSTANT_EXT:
```

Comment:
@nickdriscoll-lunarg this is all over

my advise is to update the XML as it is in https://gitlab.khronos.org/vulkan/vulkan/-/merge_requests/6906/diffs and rerun the code gen script... or just find-and-replace all the `case VK_INDIRECT_COMMANDS_TOKEN_TYPE_PUSH_CONSTANT_EXT:` that deal with getting `pPushConstant`

Reply:
I find-and-replaced all instances in custom code, but when I edited the xml and reran the generator it didn't produce different output for any generated files...